governance/cla/index.html

```diff
@@ -0,0 +1,3 @@
+---
+redirect_to: "https://github.com/libero"
```

Comment:
Link to be updated to point to the PDF?

Reply:
I can find documentation for [`redirect_from`](https://help.github.com/articles/redirects-on-github-pages/) but not `redirect_to`

Reply:
https://github.com/jekyll/jekyll-redirect-from#redirect-to